substrate/client/network/src/transport.rs

```diff
@@ -31,6 +31,9 @@ use std::{sync::Arc, time::Duration};
 
 pub use libp2p::bandwidth::BandwidthSinks;
 
+/// Timeout after which a TCP connection attempt is considered failed.
+const TCP_DIAL_TIMEOUT: Duration = Duration::from_secs(20);
```

Comment:
dq: The default timeout is in the order of minutes?

Reply:
On my linux box it's 2 min 10 secs.